app/templates/cookies/cookie_settings.html

```diff
@@ -0,0 +1,123 @@
+{% extends "_base_page.html" %}
+
+{% block pageTitle %}
+  Cookies - Digital Marketplace
+{% endblock %}
+
+{% block breadcrumb %}
+  {{ govukBreadcrumbs({"items": [
+    {
+      "text": "Digital Marketplace",
+      "href": url_for('external.index'),
+    },
+    {
+      "text": "Change your cookie settings",
+    },
+  ]}) }}
+{% endblock %}
+
+{% block mainContent %}
+
+  <div id="cookie-settings-error" class="validation-masthead cookie-settings__error">
+    <p class="validation-masthead-description">Please select 'On' or 'Off' for all cookie choices.</p>
+  </div>
+
+  <div id="cookie-settings-confirmation" class="banner-information-without-action cookie-settings__confirmation">
+    <h2 class="govuk-heading-xmedium">Your cookie settings were saved</h2>
+
+    <p class="govuk-body">You can change your preferences at any time on this page.</p>
+    <p class="govuk-body">
+        <a href="{{ url_for('external.index') }}" class="cookie-settings__prev-page">Go back to the page you were looking at.</a>
+    </p>
+
+  </div>
+
+  <div id="cookie-settings-warning" class="banner-information-without-action">
+    <h2 class="govuk-heading-xmedium">Your cookie settings have not yet been saved</h2>
+
+    <p class="govuk-body">
+        Digital Marketplace sets cookies when you visit our website.
+        You can choose to change these settings to your own preferences.
+        You need to save this page with your new choices.
+    </p>
+  </div>
+  <h1 class="govuk-heading-xl">Change your cookie settings</h1>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <p class="govuk-body">
+        Cookies are files saved on your phone, tablet or computer when you visit a website.
+      </p>
+
+      <p class="govuk-body">
+        We use cookies to store information about how you use the Digital Marketplace, such as the pages you visit.
+      </p>
+
+      <h2 class="govuk-heading-l">Cookie settings</h2>
+      <div class="cookie-settings__no-js">
+        <div class="govuk-govspeak">
+          <p>We use Javascript to set most of our cookies. Unfortunately Javascript is not running on your browser, so you cannot change your settings. You can try:</p>
+          <ul>
+            <li>reloading the page</li>
+            <li>turning on Javascript in your browser</li>
+          </ul>
+        </div>
+      </div>
+
+      <div class="cookie-settings__form-wrapper">
+        <p class="govuk-body">We use two types of cookies.</p>
+
+        <form id="cookie-settings">
+          <h3 class="govuk-heading-m">Cookies that measure website use</h3>
+
+          <p class="govuk-body">
+            We use Google Analytics to measure how you use the Digital Marketplace so we can improve it based on user needs.
```

Comment:
Please amend to: With your permission, we use Google Analytics to collect data about how you use the Digital Marketplace. This information helps us to improve our service.